aten/src/ATen/native/cpu/CatKernel.cpp

```diff
@@ -7,6 +7,14 @@
 #include <ATen/cpu/vec/vec.h>
 #include <c10/util/irange.h>
 
+// suppress warnings triggered by #pragma unroll:
```

Comment:
Same for this one. If this has been working before I dont see why this diff has to introduce these changes.